src/apps/Registration.js

```diff
@@ -56,7 +57,15 @@ const RegistrationComponent = (props) => {
                         .auth()
                         .createUserWithEmailAndPassword(email, pass)
                         .then((user) => {
-                            alert("This is where Yashika works her magick");
+                            writeData();
+                            const docRef = firebase
+                                .firestore()
+                                .collection("Users")
+                                .doc();
+                            docRef.set({
+                                Name: document.getElementById("name").value,
+                                Email: document.getElementById("email").value,
```

Comment:
```suggestion
                                email: email,
```
When the field name and value are the same, you can just say `name` instead of `name: name`. So these lines can be collapsed to `docRef.set({ name, email })`